web/server.js

```diff
@@ -325,8 +310,9 @@ module.exports = function(options) {
               otherwise: Joi.forbidden()
             })
           },
-          failAction: function(request, reply, source, error) {
-            reply(Boom.badRequest('invalid ' + source + ': ' + error.data.details[0].path));
+          failAction(request, h, error) {
+            const { source , keys: [ key ] } = error.output.payload.validation;
```

Comment:
the way we access validation failures has changed so we access `source` through a different object.